localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```
@@ -18,3 +19,186 @@
+
+ innovation_war_dinosaur:0 "War Dinsosaurs"
+ innovation_war_dinosaur_desc:0 "Once a dominant species throughout the planet, they have become an endangered remnant of an ancient world that prove to be a force to be reckoned with. These great beasts are as dangerous as they are majestic. Our tamers have successfully learned how to communicate and tame these grand creatures."
+
+ innovation_wolfrider:0 "Wolfriders"
+ innovation_wolfrider_desc:0 "These savage warriors use the speed and ferocity of their mounts to make them a deadly enemy. Faster than those who attack on foot, they have perfected the tactics of tearing through columns of enemy soldiers like a hot wind."
+ 
+ ######################
+ # Early Medieval Era #
+ ######################
+  
+ innovation_bearpaws:0 "Bearpaws"
+ innovation_bearpaws_desc:0 "Though they are only lightly armed, the bearpaws serve a big punch if provoked as strong and skilled fighters that they are."
+
+ innovation_blademaster:0 "Blademasters"
+ innovation_blademaster_desc:0 "The battle-hardened orcish Blademasters are few in number, yet they form an elite force within the Horde. Skilled swordsmen, they once belonged to the doomed clan of the Burning Blade, which, thanks to demonic corruption, is on the brink of its own destruction at all times."
+
+ innovation_baradian_marine:0 "Marines"
+ innovation_baradian_marine_desc:0 "Our finest marines not only control the seas, but are ready to stop any foe on land as well."
+    
+ innovation_chaosborn:0 "Chaosborn"
+ innovation_chaosborn_desc:0 "The chaosborn are strong casters that can both fight with magic and the sword."
+ 
+ innovation_dragknhawk:0 "Dragonhawk Taming"
+ innovation_dragknhawk_desc:0 "The dragonhawks are deft predators that rend their foes with razor-sharp talons apart. They are among the deadliest known predators, able to spot prey from great distances, rush in, and rend with both talon and beak. In addition, they're capable of breathing fire. Our tamers have successfully learned how to communicate and tame these flying creatures."
+
+ innovation_farstriders:0 "Farstriders"
+ innovation_farstriders_desc:0 "The Farstriders are an elite group of elven rangers active in Quel'Thalas. Intimately bound to the wildlands, and with a deep respect for the law, the rangers that compose the Farstriders' ranks remain noble, traditional, and valiant protectors of elven society."
+    
+ innovation_fel_guard:0 "Felguards"
+ innovation_fel_guard_desc:0 "Felguard are fearless and unwaveringly loyal to their master's godlike will.  They are almost always seen with heavy, two-handed weaponry. They are slow to strike, but hit quite hard, and are more agile than they look."
+    
+ innovation_footmen:0 "Footmen"
+ innovation_footmen_desc:0 "Footmen are warriors and versatile foot soldiers who serve as basic trained infantry in armies of the human nations, being much more well equipped than their counterparts in other armies."
```

I'm not sure about this sentence "Well equipped than their counterparts in other armies.", changing that:
```suggestion
 innovation_footmen_desc:0 "Footmen are warriors and versatile foot soldiers who serve as basic trained infantry in armies of the human nations. Economical yet effective, they are cladded in heavy armor and hefty armaments."
```